### ISSUE TITLE
Wrong Attitude angles 

### ISSUE THREAD

**Issue by droneconnect** [BluDay/wiiproxy#4]
I am just checking to output the controller.get_attitude(False)) function and getting the following:

{'angx': 1382, 'angy': 383, 'heading': 22}
{'angx': 1382, 'angy': 383, 'heading': 22}
{'angx': 1383, 'angy': 383, 'heading': 22}
{'angx': 1383, 'angy': 383, 'heading': 22}
{'angx': 1384, 'angy': 383, 'heading': 22}
{'angx': 1385, 'angy': 382, 'heading': 22}
{'angx': 1386, 'angy': 382, 'heading': 22}
{'angx': 1388, 'angy': 382, 'heading': 22}
{'angx': 1389, 'angy': 382, 'heading': 21}
{'angx': 1390, 'angy': 381, 'heading': 21}
{'angx': 1391, 'angy': 381, 'heading': 21}

the test has been done with boards running STM32F4 and also the old NAZE32..
only the heading goes up to 360, X and Y go beyond the 360 limit....

**Comment by bluday** [comment]
@droneconnect I accidentally forgot to divide X and Y with 10.0 as stated in the MSP documentation.

It has been fixed and resolved and I've also added new functionalities to the module.

A status update would be greatly appreciated...

**Comment by droneconnect** [comment]
very good , it looks better on the Attitude, but on the controller.get_imu(False)) now there is a problem.
The accz looks off, on a constant value of 500. I tested with 2 different boards and getting the following data:

{'accx': 5.0, 'accy': 17.0, 'accz': 511.0, 'gyrx': -1.0, 'gyry': -2.0, 'gyrz': 0.0, 'magx': 0.0, 'magy': 0.0, 'magz': 0.0}
{'accx': 5.0, 'accy': 16.0, 'accz': 511.0, 'gyrx': -4.0, 'gyry': 3.0, 'gyrz': 3.0, 'magx': 0.0, 'magy': 0.0, 'magz': 0.0}
{'accx': 5.0, 'accy': 16.0, 'accz': 511.0, 'gyrx': -6.0, 'gyry': 1.0, 'gyrz': 1.0, 'magx': 0.0, 'magy': 0.0, 'magz': 0.0}
{'accx': 5.0, 'accy': 16.0, 'accz': 511.0, 'gyrx': -3.0, 'gyry': 0.0, 'gyrz': 0.0, 'magx': 0.0, 'magy': 0.0, 'magz': 0.0}
{'accx': 5.0, 'accy': 15.0, 'accz': 511.0, 'gyrx': 18.0, 'gyry': -11.0, 'gyrz': -1.0, 'magx': 0.0, 'magy': 0.0, 'magz': 0.0}
{'accx': 5.0, 'accy': 15.0, 'accz': 511.0, 'gyrx': 22.0, 'gyry': -13.0, 'gyrz': -3.0, 'magx': 0.0, 'magy': 0.0, 'magz': 0.0}


this with the board standing still on the table

**Comment by bluday** [comment]
@droneconnect I've tried to find out exactly why this occurs and I was not able to resolve this issue.

The MultiWii Serial Protocol unfortunately has a poor documentation with unclear statements.

The comment of MSP_RAW_IMU does infact state an unspecific unit that each angle needs to be converted to.

I also checked the ACC angles on the MultiWiiConf software and found out that my "accz" angle also is around 500. I'm uncertain whether this issue possibly has to do with my configuration files (config.h, def.h) on the MultiWii firmware.
